tsweb/tsweb.go

```diff
@@ -196,8 +196,8 @@ type BucketedStatsOptions struct {
 // normalizePathRegex matches components in a HTTP request path
 // that should be replaced.
 //
-// See: https://regex101.com/r/WIfpaR/1 for the explainer and test cases.
-var normalizePathRegex = regexp.MustCompile("([a-fA-F0-9]{9,}|([^\\/])+\\.([^\\/]){2,}|((n|k|u|L|t|S)[a-zA-Z0-9]{5,}(CNTRL|Djz1H|LV5CY|mxgaY|jNy1b)))")
+// See: https://regex101.com/r/WIfpaR/3 for the explainer and test cases.
+var normalizePathRegex = regexp.MustCompile("([a-fA-F0-9]{9,}|([^\\/])+\\.([^\\/]){2,}|((n|k|u|L|t|S)[a-zA-Z0-9]{5,}(CNTRL|Djz1H|LV5CY|mxgaY|jNy1b))|(([^\\/])+\\@passkey))")
```

Comment:
FWIW, if you want to remove some of the double-escaping you could use backtick strings? Up to you though.
```suggestion
var normalizePathRegex = regexp.MustCompile(`([a-fA-F0-9]{9,}|([^\/])+\.([^\/]){2,}|((n|k|u|L|t|S)[a-zA-Z0-9]{5,}(CNTRL|Djz1H|LV5CY|mxgaY|jNy1b))|(([^\/])+\@passkey))`)
```

Reply:
Ooo I forgot about this! Will defs use in the next iteration